test/controllers/browse_controller_test.rb

```diff
@@ -60,33 +60,27 @@
 
       describe "with the new navigation enabled" do
         it "returns the original version of the page as the A variant" do
-          with_new_navigation_enabled do
-            with_variant EducationNavigation: "A" do
-              get :show, top_level_slug: "education"
+          with_new_navigation_disabled do
```

Comment:
This contradicts the `describe` statement on line 61. I think it's just a case of wording - perhaps this should just be called `with_A_variant`?